Latest Docker image throws: Error: Cannot find module '@babel/polyfill'

#### Description

Describe the issue or feature request in detail. Take a screenshot or record a video will be helpful.

#### Versions

- CNCjs: 1.10.2 (Docker latest)
- Node.js: 6.x
- NPM: 5.x

#### How Do You Install CNCjs?

- [ ] NPM
- [ ] Download the CNCjs Desktop Application
- [x] Docker

#### CNC Controller

- [x] Grbl
- [ ] Smoothieware
- [ ] TinyG/g2core

#### Hardware

- [ ] Raspberry Pi
- [x] Desktop or Laptop
- [ ] Mobile Device

#### Operating System

- [ ] Not Applicable
- [x] Windows
- [ ] Mac
- [x] Linux

I am using the cncjs:latest docker container. 
(Also present in older images)
The container throws following error on start:

`internal/modules/cjs/loader.js:905
throw err;
^

Error: Cannot find module '@babel/polyfill'
Require stack:

    /opt/cncjs/entrypoint
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object. (/opt/cncjs/entrypoint:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12) {
    code: 'MODULE_NOT_FOUND',
    requireStack: [ '/opt/cncjs/entrypoint' ]
    }`

Does the Dockerimage use an old CNCJS version?

I can confirm that the latest image for me has this error, running v1.10.2:
```
root@buffalo:/srv/cncjs# docker ps | grep cncjs
4836893bc876   cncjs/cncjs:v1.10.2             "/opt/cncjs/entrypoi…"   3 hours ago   Restarting (1) 12 seconds ago                                                                                                                     cncjs-cnc-1
```

Docker Log:
```
Error: Cannot find module '@babel/polyfill'
Require stack:
- /opt/cncjs/entrypoint
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/opt/cncjs/entrypoint:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/opt/cncjs/entrypoint' ]
}
internal/modules/cjs/loader.js:905
  throw err;
  ^
```
The above repeats over and over as the container restarts.

--- a/entrypoint
+++ b/entrypoint
@@ -1,8 +1,6 @@
 #!/usr/bin/env node
 
-require('@babel/polyfill');
-
-const launchServer = require('/opt/cncjs/server-cli').default;
+const launchServer = require('/opt/cncjs/server-cli');
 
 launchServer().catch(err => {
   console.error('Error:', err);

